electron/src/webview/index.ts

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+import { ipcRenderer } from "electron";
+import { WebviewStartMessage } from "../typings/interfaces";
+
+function start(event: Electron.IpcRendererEvent, message: WebviewStartMessage) {
+    if (message.devtools) {
+        webview.addEventListener("did-stop-loading", () => {
+            webview.openDevTools();
+        });
+    }
+
+    webview.src = `http://localhost:3232/#local_token=${message.localtoken}&electron=1`;
```

Comment:
Now that the CLI supports custom port I'll add logic to find a free port in the Electron main process, and then it can communicate the port to the webview